docs/docs/sourcing-from-yaml.md

```diff
@@ -0,0 +1,232 @@
+---
+title: "YAML as a source for building a site"
+---
+
+# Table of Contents
+
+- [Introduction](#Introduction)
+- [Setup](#Setup)
+- [Adding Content](#Adding-some-content)
+- [Gatsby Config](#Gatsby-configuration)
+- [Gatsby Template](#Gatsby-Template)
+- [Joining the pieces](#Joining-the-pieces)
+- [Sourcing from JSON](https://www.gatsbyjs.org/docs/using-gatsby-without-graphql/)
```

Comment:
We should make links match their URLs wherever possible:
```suggestion
- [Using Gatsby without GraphQL](https://www.gatsbyjs.org/docs/using-gatsby-without-graphql/)
```

Reply:
Changed the title accordingly to match the existing item